compliance/azure/azure_untagged_resources/untagged_resources.pt

```diff
@@ -2,11 +2,12 @@ name "Azure Untagged Resources"
 rs_pt_ver 20180301
```

Comment:
is it possible to rename this file `azure_untagged_resources.pt` ? Not sure of the implications on any documentation but I think it should be okay